.travis.yml

```diff
@@ -1,5 +1,6 @@
 rvm:
-  - "2.0.0-p594"
-  - "2.1.4"
-  
+  - "2.0.0-p451"
```

Comment:
Maybe just drop the patch level altogether here? Might help avoid this going forward.